test/core/security/security_connector_test.cc

```diff
@@ -664,9 +664,12 @@ static void test_default_ssl_roots(void) {
   fwrite(roots_for_env_var, 1, strlen(roots_for_env_var), roots_env_var_file);
   fclose(roots_env_var_file);
 
+  grpc_core::ConfigVars::Overrides overrides;
+
   // First let's get the root through the override: set the env to an invalid
```

Comment:
Please update these comments that talk about env vars (various places in this file).

Reply:
done